docs/install/install-script.md

```diff
@@ -87,3 +87,9 @@ To wipe out the ModelMesh Serving CRDs, controller, and built-in runtimes from t
 ```shell
 ./scripts/delete.sh --namespace modelmesh-serving
 ```
+
+(Optional) Delete the specify namespace `modelmesh-serving`
```

Comment:
Nit, consider "specified" instead of "specify"

Reply:
Good catch, @chinhuang007 , I will update it, Thanks

Reply:
Updated, Thanks